cli/slsa-verifier/main.go

```diff
@@ -41,21 +43,25 @@ func main() {
 		os.Exit(1)
 	}
 
-	var ptag, pversiontag *string
+	var pbuilderID, ptag, pversiontag *string
 
 	if isFlagPassed("tag") {
 		ptag = &tag
 	}
 	if isFlagPassed("versioned-tag") {
 		pversiontag = &versiontag
 	}
+	if isFlagPassed("builder-id") {
+		pbuilderID = &builderID
+	}
```

Comment:
aside: I'm curious. Remind me again why these flags are copied to local variables? Also, why we need a special `isFlagPassed` function?

Reply:
I don't think I found a way to have an option be a pointer to a string, and for the pointer to be nil if the user does not provide the option. If that's possible, I can update the code.